src/main/scala/workflow/graph/Identity.scala

```diff
@@ -0,0 +1,15 @@
+package workflow.graph
+
+import org.apache.spark.rdd.RDD
+
+import scala.reflect.ClassTag
+
+/**
+ * This transformer performs a no-op on its input.
+ *
+ * @tparam T Type of the input and, by definition, output.
+ */
+case class Identity[T : ClassTag]() extends Transformer[T,T] {
```

Comment:
Similar to cacher - suddenly users have to implement different methods - it's not clear to me why this naming has to change.